### PR TITLE
Feature/greeting farewell mini expansion

### LINKED ISSUE #696: [ FEATURE REQUEST ] - Greeting, farewell mini-expansion
# Overview:
Currently, we have a decent range of new greetings and farewells: however, over time these can feel stale - particularly to long-time players, or those still working on a certain affinity.

We can expand these a little to add a bit more variety and freshness when booting up the mod and saying goodbye to Natsuki.

It should be noted this is aimed to be introduced in the upcoming 1.2 patch, so should be limited in scope and not include any large, sweeping/complicated items: added greetings/farewells should be relatively simple in nature (no complex dialogue trees).

## Must haves:
- One new greeting for each tier
- One new farewell for each tier

## Could haves:
- Additional greetings as appropriate
- Additional farewells as appropriate

## Is this feature request related to a problem? Please describe.
This is related to the existing greeting/farewell functionality.

### PR DESCRIPTION
**Pull request by Blizzardsev**
Introduces the following:

- Seven new greetings:
    - `greeting_love_plus_nat_dragged_in`
    - `greeting_love_plus_show_yourself`
    - `greeting_affectionate_enamored_not_surprised`
    - `greeting_affectionate_enamored_in_for_some_fun`
    - `greeting_normal_oh_whats_up`
    - `greeting_distressed_upset_oh_great`
    - `greeting_broken_minus_just_leave_me_alone`

- Six new farewells:
    - `farewell_love_that_time_again`
    - `farewell_affectionate_enamored_making_it_up`
    - `farewell_happy_affectionate_catch_you_later`
    - `farewell_normal_happy_oh_right`
    - `farewell_upset_distressed_yeah_bye`
    - `farewell_broken_ruined_door_hit_you`

Closes #696

### CODE REVIEW

File: game/script-greetings.rpy [hunk 1]
```diff
@@ -874,6 +874,49 @@ label greeting_love_plus_dull_moment:
...
+        ),
+        topic_group=TOPIC_TYPE_GREETING
+    )
+
+label greeting_love_plus_cant_get_enough:
+    n 4ccsbg "Well,{w=0.2} well,{w=0.2} well.{w=0.75}{nw}"
+    extend 3fsqbg " And just look who decided to show up today.{w=0.75}{nw}"
+    extend 3fsqsm " Ehehe."
+    n 1fcsbgl "Well,{w=0.2} what can I say?{w=0.75}{nw}"
+    extend 2fchgnl " Guess you really {i}can't{/i} get enough of me,{w=0.2} huh?"
```
Comment (RaionArt) [inline game/script-greetings.rpy:893]: I noticed this greeting is really similar to an already existing one. To be precises the `greeting_affectionate_enamored_just_cant_stay_away` greeting. I don't think it is bad to have reappearing speak patterns, but I don't see something that justify love affinity, when we already have something similar for lower affinity levels than that.

**Review by RaionArt — CHANGES_REQUESTED** [review]
Overall I think this is a great addition to give Natsuki more variety on a dialy basis! While the player can hopefully also notice new greetings/farewells when getting to a new affinity tier. 
I only had a bit of a concern to one of the greetings.

File: game/script-greetings.rpy (hunk 1 above)
Comment (Blizzardsev) [reply game/script-greetings.rpy:893]: Sure, I can adjust this to be a little more deserving of the affinity.

Comment (Blizzardsev) [reply game/script-greetings.rpy:893]: The only thing I'd note with the existing greeting: the affinity ranges are pretty tight on these, so a player on LOVE+ affinity would only get this new one, and would no longer see the `greeting_affectionate_enamored_just_cant_stay_away` greeting (as this stops being applicable at ENAMORED).

Comment (Blizzardsev) [reply game/script-greetings.rpy]: This has been resolved, reworked the greeting into something a bit more distinct: `greeting_love_plus_nat_dragged_in`

**Review by RaionArt — APPROVED** [review]
The _Nat_ is my new favourite, good job

**Pull request merged**